macros/src/vertex.rs

```diff
@@ -59,7 +59,7 @@ fn body(ecx: &mut base::ExtCtxt, span: codemap::Span,
                 .map(|(def, &(ident, _))| {
                     let ref elem_type = def.node.ty;
                     let ident_str = token::get_ident(ident);
-                    let ident_str = ident_str.get();
+                    let ident_str = ::std::ops::Deref::deref(&ident_str);
```

Comment:
Fixed :)
Since I'm fixing this blindly, I went for the syntax I know will work.